lib/ts/recipe/oauth2provider/index.ts

```diff
@@ -68,6 +68,13 @@ export default class Wrapper {
             userContext: getUserContext(userContext),
         });
     }
+
+    static revokeToken(token: string, userContext?: Record<string, any>) {
```

Comment:
Shouldn't this take authorization info as well?

Reply:
Fixed